src/reducers/url-state.js

```diff
@@ -230,6 +268,15 @@ const hiddenGlobalTracks: Reducer<Set<TrackIndex>> = (
       hiddenGlobalTracks.delete(action.trackIndex);
       return hiddenGlobalTracks;
     }
+    case 'HIDDEN_GLOBAL_TRACKS_REMOVED': {
+      // At this point we don't know if all or some of the tracks are removed.
+      // That's why we are iterating through the hiddenTracks and deleting them.
+      const hiddenGlobalTracks = new Set(state);
+      for (const hiddenThread of action.hiddenTracks) {
+        hiddenGlobalTracks.delete(hiddenThread);
+      }
+      return hiddenGlobalTracks;
+    }
     default:
       return state;
   }
```

Comment:
I'm not convinced that we should change the current view. If I remember our previous discussions well we wanted to show the permalink to the newly uploaded profile, and make it easy to open a new page with the new uploaded profile instead. Or maybe open it directly as a first easier step.